go/test/endtoend/cluster/vtctld_process.go

```diff
@@ -58,15 +58,25 @@ func (vtctld *VtctldProcess) Setup(cell string, extraArgs ...string) (err error)
 		"--topo_global_server_address", vtctld.CommonArg.TopoGlobalAddress,
 		"--topo_global_root", vtctld.CommonArg.TopoGlobalRoot,
 		"--cell", cell,
-		"--workflow_manager_init",
-		"--workflow_manager_use_election",
 		"--service_map", vtctld.ServiceMap,
 		"--backup_storage_implementation", vtctld.BackupStorageImplementation,
 		"--file_backup_storage_root", vtctld.FileBackupStorageRoot,
 		"--log_dir", vtctld.LogDir,
 		"--port", fmt.Sprintf("%d", vtctld.Port),
 		"--grpc_port", fmt.Sprintf("%d", vtctld.GrpcPort),
 	)
+	var majorVersion int
+	majorVersion, err = GetMajorVersion("vtctld")
+	if err != nil {
+		return err
+	}
+	// workflow_manager_init and workflow_manager_use_election are removed in v16 and shouldn't be set on any release v16+
+	if majorVersion <= 15 {
+		vtctld.proc.Args = append(vtctld.proc.Args,
+			"--workflow_manager_init",
+			"--workflow_manager_use_election")
+	}
+
```

Comment:
None of the endtoend tests actually ever tested workflow manager. We don't need this conditional code at all.

Reply:
Done